readme.txt

```diff
@@ -239,7 +239,9 @@ Remember to always make a backup of your database and files before updating!
 * Fix - Avoid the issue of events which have venues assigned not being able to be updated successfuly on some browsers. [TEC-4596]
 * Fix - Handle the case where rewrite rules map to arrays avoiding fatal errors. [TEC-4567]
 * Fix - Fix a primary cause of MySQL `Deadlock` errors in 6.0 event migration and added `Deadlock` error catching in our lock/fetch event queue. [TEC-4548]
+* Fix - Ensure we did not get an error object back when requesting event category. [TEC-4619]
 * Tweak - Convert all uses of (view)->get_slug() to (view)::get_view_slug(). [TEC-4586]
+* Tweak - Change some labelling of event settings in the admin. [TEC-4626]
```

Comment:
nothing to see here 🙈 just an entry that got left off earlier

move along. 

Go on now! Shoo!